packages/synapse-react-client/src/lib/containers/auth/Login.tsx

```diff
@@ -0,0 +1,211 @@
+import * as React from 'react'
+import { Form } from 'react-bootstrap'
+import ButtonWithIcon from '../../assets/ButtonWithIcon'
+import { SynapseClient } from '../../utils'
+import {
+  getEndpoint,
+  BackendDestinationEnum,
+} from '../../utils/functions/getEndpoint'
+import { GoogleIcon24 } from '../../assets/GoogleIcon24'
+import { Button, IconButton, Link } from '@mui/material'
+import IconSvg from '../IconSvg'
+import FullWidthAlert from '../FullWidthAlert'
+import { useState } from 'react'
+
+export const PROVIDERS = {
+  GOOGLE: 'GOOGLE_OAUTH_2_0',
+  ORCID: 'ORCID',
+}
+
+type Props = {
+  ssoRedirectUrl?: string
+  sessionCallback: () => void // Callback is invoked after login
```

Comment:
We had an additional prop `redirectUrl` but it was never used internally, so it was dropped.